setup.cfg

```diff
@@ -31,27 +31,27 @@ install_requires =
     Flask-Cors~=3.0
     Flask-Mail~=0.9
     Flask-Migrate>=2.5.3,<4  # Not following semantic versioning (e.g. https://github.com/miguelgrinberg/flask-migrate/commit/1af28ba273de6c88544623b8dc02dd539340294b)
-    Flask-RESTful~=0.3
+    Flask-RESTful~=0.3,>=0.3.9
```

Comment:
Can I suggest:
```suggestion
    Flask-RESTful>=0.3.9,<0.4.0
```

Reply:
According to the table in https://pip.pypa.io/en/stable/user_guide/#fixing-conflicting-dependencies `~=0.3,>=0.3.9` is equivalent to `~=0.3.9`

Reply:
From my understanding, `~=0.3,>=0.3.9` allows `0.4.X` while `~=0.3.9` does not

Reply:
> Can I suggest:

The current requirement is a bit different, it's equivalent to:

`Flask-RESTful>=0.3.9,<1.0.0`

which makes sense for semantic versioning.

But in general, I'm all for getting rid of this confusing `~=` requirement, it's too implicit.